src/index.js

```diff
@@ -50,6 +52,39 @@ export const compose = (...funcs) => {
 
 export const createMediaQuery = n => `@media screen and (min-width: ${px(n)})`
 
+const getStyles = (props, style, val) => {
```

Comment:
For some reason, it feels like pulling this out from the other function makes it a little harder to compare with the previous implementation and don’t see much re-use in the `getSttyles` function – would you mind putting this back in the main function body?

Reply:
Sure no problem. Will result in a little bit of duplication across top-level `space` and `style`  fns (~30 LoC).

Reply:
Ah, I see now. Yeah, `space` is an odd-ball, and I'd like to try to get rid of that (in favor of using `style`) in the next major version – it's probably fine to leave it split out then, but could be nice to use a single object argument like the other functions do